pkg/client/unversioned/record/event.go

```diff
@@ -222,6 +222,9 @@ func (eventBroadcaster *eventBroadcasterImpl) StartEventWatcher(eventHandler fun
 				// ever happen.
 				continue
 			}
+			if ratelimiter != nil {
+				ratelimiter.Accept()
```

Comment:
We have a default event queue with the length of 1000 and implicit max event limit is 5 QPS with no extra rate limiter. It works normally now. Suppose 1000 length of queue and 5 QPS can handle heavy traffic for at most 30s, when rate limit is 1QPS, the performance shoule be the same with about 1000+(5QPS-1QPS)＊30S length of queue(need more 120 length of queue).   But I don't think we need to increase length of queue.